File: Day 1/Python/TaskNine.py


Score_Counter = 0
sum_Of_Scores = 0
number_Of_Students = 10
even_Number = 2
sum_Of_Even_Numers = 0
number_of_Even_Numbers = 0

while Score_Counter < 10:

	score = int(input("Enter a score: "))
	if score < 1 or score > 100:
		print("Enter a valid score")
		Score_Counter = Score_Counter - 1
	else:
		sum_Of_Scores  += score;

	Score_Counter = Score_Counter + 1

	if score % even_Number == 0:
		sum_Of_Even_Numers += score
		number_of_Even_Numbers = number_of_Even_Numbers + 1




average = sum_Of_Scores / number_Of_Students


print(f'The sum of numbers is {sum_Of_Scores}');

File: Day 1/Java/TaskNine.java
import java.util.Scanner;
public class TaskEight{
public static void main(String[] args){

Scanner input = new Scanner(System.in);

int ScoreCounter = 0;
int sumOfScores = 0;
int numberOfStudents = 10;
int evenNumber = 2;
int sumOfEvenNumbers = 0;
int numberOfEvenNumbers = 0;

while (ScoreCounter < 10){

System.out.print("Enter a score: ");

int score = input.nextInt();
if (score < 1 || score > 100){

System.out.print("Enter valid score\n");

-- ScoreCounter;
}
 

else {
sumOfScores += score;
}

ScoreCounter++;

if(score % evenNumber == 0){
 sumOfEvenNumbers += score;
numberOfEvenNumbers ++;
}
}

double average = sumOfEvenNumbers / numberOfEvenNumbers;


System.out.print("\nThe sum of the numbers is "+ sumOfScores);



}
}